server/models/Request.model.js

```diff
@@ -0,0 +1,19 @@
+const mongoose = require("mongoose");
+
+const requestSchema = new mongoose.Schema(
+  {
+    user: { type: mongoose.ObjectId, required: true, ref: "user" },
+    sitter: { type: mongoose.ObjectId, required: true, ref: "user" },
+    dog: { type: mongoose.ObjectId, required: true, ref: "dog" },
+    start: { type: Date, required: true, trim: true },
+    end: { type: Date, required: true, trim: true },
+    accepted: { type: Boolean },
+    declined: { type: Boolean },
+    paid: { type: Boolean },
```

Comment:
Just to give a heads up, you can use the shorthand notation for `type` if you wanted to (example [here](https://mongoosejs.com/docs/guide.html#definition)).